### ISSUE TITLE
Lua runtime error if token header without spaces is provided (regression since 1.7.6)

### ISSUE THREAD

**Issue by konstantin-smolyakov-idt** [zmartzone/lua-resty-openidc#473]
###### Environment

- lua-resty-openidc version: 1.7.6-3

###### Expected behaviour

A token validation error is returned (resulting in 401 response).

###### Actual behaviour

A runtime error occurs and 500 response is returned:

```
/usr/local/share/lua/5.1/resty/openidc.lua:1654: attempt to perform arithmetic on local 'divider' (a nil value)
```

###### Minimized example

Send a bearer token without spaces, e.g. like this

```
Authorization: Bearer12345
```

**Comment by bodewig** [comment]
thank you, fixed with ae3bc54

**Comment by rjbma** [comment]
Has this been released, and if not, are there any plans to release it?

I'm currently hitting this bug, but latest released version (1.7.6-3) doesn't seem to include this fix yet.
